lib/matplotlib/tests/test_axes.py

```diff
@@ -3794,6 +3794,22 @@ def test_horiz_violinplot_custompoints_200():
                   showextrema=False, showmedians=False, points=200)
 
 
+@image_comparison(['violinplot_sides.png'])
+def test_violinplot_sides():
+    ax = plt.axes()
+    # First 9 digits of frac(sqrt(43))
+    np.random.seed(557438524)
```

Comment:
```suggestion
    np.random.seed(19680801)
```

Admittedly tests near here do not follow our guidance on random data, but we also have a standard for that:

https://matplotlib.org/devdocs/devel/testing.html#random-data-in-tests

It's not particularly important or anything (that a seed is set is more important than the value of the seed) but I'd lean towards the written standard seed for new tests.